pkgs/development/compilers/open-watcom/v2.nix

```diff
@@ -120,7 +120,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = "https://open-watcom.github.io";
     license = licenses.watcom;
-    platforms = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "x86_64-windows" "i686-windows" ];
+    platforms = with platforms; arm ++ aarch64 ++ x86 ++ mips; # ++ powerpc
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
No, just Unix in general really isn't right… If this is too specific for you then I'll compromise by just marking some known-incompatible platforms in `badPlatforms`.